gradyl16/maze3
# Peter AI transport

AI transport, player pickup / dropoff, game fail and level complete, all should be done.

To test, open the:
Assets/Scenes/Individual/Peter/TransportEnemyTest scene

Pickup and drop-off with E key, when close to the right object.

Expected behavior:
- if "wanted" level is 0, enemies just transport boxes around
- if "wanted" level > 0, enemies will chase after player
- wanted level starts with 0, increase by one with each objective completion
- to complete objective, deliver colored box to the different colored drop-off
- if box delivered to wrong location, smoke should show up at that location and all enemies will be hostile
- if box delivered to right location (same color), then wanted level doesn't increase
- if box delivered to wrong location that already has smoke, wanted level doesn't change
- player dies if comes in contact with enemy
- player finishes level if all objectives destroyed / have smoke

Approving this PR and mergin to main will (most likely) break the existing scenes!
Remove all camera and player objects and use the Assets/Prefabs/Player prefab instead.

To add my enemies to any scene:
- check the comment at the top of Assets/Scripts/Enemy/EnemyTransportWorker.cs script
